src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -371,13 +384,28 @@ const renderPricing = (
     const now = DateTime.local()
     const awaitingResults = dateOfSale > now
 
+    const showPriceUSD = salePriceUSD && currency !== "USD"
+
     return (
       <Box textAlign={textAlign} mb="5px">
         {salePrice && (
           <>
-            <Text variant="md" fontWeight="bold">
-              {salePrice}
-            </Text>
+            <Flex alignItems="center" justifyContent="flex-end">
+              <Text variant="md" fontWeight="bold">
+                {salePrice}
+              </Text>
+              {!!showPriceUSD && (
+                <Text
+                  variant="xs"
+                  lineHeight={1}
+                  color="black60"
+                  style={{ paddingTop: "2px" }}
```

Comment:
Couldn't find a way a better way to have `salePrice` and `salePriceUSD` be aligned correctly without applying top padding. Any ideas?

Reply:
Only suggestion would be to remove the `lineHeight` prop above, as that might be throwing off the `alignItems=center` on the flex container; those values are baked into the variants and shouldn't be adjusted if possible. 

@dzucconi might have some suggestions!

Reply:
Setting `lineHeight={1}` on both should work fine.

Reply:
Thanks for the help! Adding `alignItems="flex-end"` to the parent container did the trick 🎉